sbc_switzerland/models/correspondence.py

```diff
@@ -124,6 +124,11 @@ def process_letter(self):
                     letter.has_valid_language:
                 if super(Correspondence, letter).process_letter():
                     letters_to_send += letter
+                    intro_letter = self.env.ref(
+                        'sbc_compassion.correspondence_type_new_sponsor')
+                    if intro_letter in letter.communication_type_ids and not\
```

Comment:
This test should be placed before line 126 otherwise the letter will still be sent.